test/juttled/juttled.spec.js

```diff
@@ -440,6 +440,84 @@ describe('Juttled Tests', function() {
         });
     });
 
+    describe('Get Directory Tests', function() {
+        describe('Valid cases', function() {
+
+            it('list contents of current directory', function() {
+                return chakram.get(jd + '/directory')
+                .then(function(response) {
+                    expect(response).to.have.status(200);
+
+                    var body = response.body;
+                    expect(body.is_root).to.be.true;
+                    expect(body.path).to.equal('/');
+                    expect(body.directories).to.deep.equal(['subdir']);
+                    expect(body.juttles.length).to.be.above(0)
+                })
+            });
+
+            it('list contents of subdir', function() {
+                return chakram.get(jd + '/directory?path=subdir')
+                .then(function(response) {
+                    expect(response).to.have.status(200);
+                    expect(response.body.is_root).to.be.false;
+                    expect(response.body.path).to.equal('/subdir');
```

Comment:
I'd suggest also checking for no directories below subdir and >0 files below subdir.